src/Uno.UI/Microsoft/UI/Xaml/Controls/ColorPicker/ColorSpectrum.cs

```diff
@@ -0,0 +1,1807 @@
+					//	}
+					//	else if (sextant >= 3 && sextant < 4)
+					//	{
+					//		spectrumBrush.MinSurface(m_hueCyanSurface);
+					//		spectrumBrush.MaxSurface(m_hueBlueSurface);
+					//	}
+					//	else if (sextant >= 4 && sextant < 5)
+					//	{
+					//		spectrumBrush.MinSurface(m_hueBlueSurface);
+					//		spectrumBrush.MaxSurface(m_huePurpleSurface);
+					//	}
+					//	else
+					//	{
+					//		spectrumBrush.MinSurface(m_huePurpleSurface);
+					//		spectrumBrush.MaxSurface(m_hueRedSurface);
+					//	}
+
+					//	spectrumBrush.MaxSurfaceOpacity(sextant - (int)sextant);
+					//	spectrumRectangle.Fill = spectrumBrush;
+					//	spectrumEllipse.Fill = spectrumBrush;
+					//}
+					//else
+					{
+						if (m_hueRedBitmap == null ||
+							m_hueYellowBitmap == null ||
+							m_hueGreenBitmap == null ||
+							m_hueCyanBitmap == null ||
+							m_hueBlueBitmap == null ||
+							m_huePurpleBitmap == null)
+						{
+							return;
+						}
+
+						ImageBrush spectrumBrush = new ImageBrush();
+						ImageBrush spectrumOverlayBrush = new ImageBrush();
+
+						double sextant = Hsv.GetHue(hsvColor) / 60.0;
+
+						if (sextant < 1)
+						{
+							spectrumBrush.ImageSource = m_hueRedBitmap;
+							spectrumOverlayBrush.ImageSource = m_hueYellowBitmap;
+						}
+						else if (sextant >= 1 && sextant < 2)
+						{
+							spectrumBrush.ImageSource = m_hueYellowBitmap;
+							spectrumOverlayBrush.ImageSource = m_hueGreenBitmap;
+						}
+						else if (sextant >= 2 && sextant < 3)
+						{
+							spectrumBrush.ImageSource = m_hueGreenBitmap;
+							spectrumOverlayBrush.ImageSource = m_hueCyanBitmap;
+						}
+						else if (sextant >= 3 && sextant < 4)
+						{
+							spectrumBrush.ImageSource = m_hueCyanBitmap;
+							spectrumOverlayBrush.ImageSource = m_hueBlueBitmap;
+						}
+						else if (sextant >= 4 && sextant < 5)
+						{
+							spectrumBrush.ImageSource = m_hueBlueBitmap;
+							spectrumOverlayBrush.ImageSource = m_huePurpleBitmap;
+						}
+						else
+						{
+							spectrumBrush.ImageSource = m_huePurpleBitmap;
+							spectrumOverlayBrush.ImageSource = m_hueRedBitmap;
+						}
+
+						spectrumOverlayRectangle.Opacity = sextant - (int)sextant;
+						spectrumOverlayEllipse.Opacity = sextant - (int)sextant;
+						spectrumRectangle.Fill = spectrumBrush;
+						spectrumEllipse.Fill = spectrumBrush;
+						spectrumOverlayRectangle.Fill = spectrumOverlayBrush;
+						spectrumOverlayRectangle.Fill = spectrumOverlayBrush;
+					}
+					break;
+			}
+		}
+
+		private bool SelectionEllipseShouldBeLight()
+		{
+			// The selection ellipse should be light if and only if the chosen color
+			// contrasts more with black than it does with white.
+			// To find how much something contrasts with white, we use the equation
+			// for relative luminance, which is given by
+			//
+			// L = 0.2126 * Rg + 0.7152 * Gg + 0.0722 * Bg
+			//
+			// where Xg = { X/3294 if X <= 10, (R/269 + 0.0513)^2.4 otherwise }
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=6121259)